app/src/main/java/com/androidtitan/culturedapp/main/no_internet/NoInternetAdapter.java

```diff
@@ -0,0 +1,76 @@
+package com.androidtitan.culturedapp.main.no_internet;
+
+import com.androidtitan.culturedapp.R;
+import com.androidtitan.culturedapp.common.CollectionUtils;
+import com.androidtitan.culturedapp.databinding.NoInternetListitemBinding;
+import com.androidtitan.culturedapp.model.newyorktimes.Article;
+import android.content.Context;
+import android.databinding.DataBindingUtil;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+
+import java.util.List;
+
+/**
+ * Created by amohnacs on 9/6/17.
+ */
+
+public class NoInternetAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+    private static final String TAG = NoInternetAdapter.class.getSimpleName();
+
+    private Context context;
+    private List<Article> offlineArticles;
+
+    public NoInternetAdapter(Context context, List<Article> offlineArticles) {
+        this.context = context;
+        this.offlineArticles = offlineArticles;
+    }
+
+    @Override
+    public RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+
+        LayoutInflater layoutInflater = LayoutInflater.from(context);
+        NoInternetListitemBinding listBinding = DataBindingUtil.inflate(layoutInflater,
+                R.layout.no_internet_listitem, parent, false);
```

Comment:
This class holds all the bindings from the layout properties (e.g. the user variable) to the layout's Views and knows how to assign values for the binding expressions.The easiest means for creating the bindings is to do it while inflating